packages/pigeon/lib/dart_generator.dart

```diff
@@ -12,7 +12,11 @@ import 'functional.dart';
 import 'generator_tools.dart';
 
 /// Documentation comment open symbol.
-const String _docCommentPrefix = '/// ';
+const String _docCommentPrefix = '///';
```

Comment:
Optional: you could just inline this since we don't expect to ever use a documentation comment marker except for with `addDocumentationComments` (unlike the C++ version where it's the same as a normal comment).

Reply:
I was going for consistency between generator files. I'll defer to you on preference though.

Reply:
> I was going for consistency between generator files.

Makes sense.